agent/wpthook/test_state.cc

```diff
@@ -349,7 +363,7 @@ bool TestState::IsDone() {
         _test_result = TEST_RESULT_TIMEOUT;

```

Comment:
Yes, but that's what we want. This step is for minimum duration, so it does additional checks to see if we should time out.

We want it to timeout if the whole test has timed-out or if a specific-step times out. If a specific step times out, we'll still continue counting the overall-test timeout value for the next step.